_posts/2020-12-20-azure-event-grid.md

```diff
@@ -0,0 +1,55 @@
+- eventi "atomici": ovvero eventi che hanno la loro ragione di essere completamente scorrelata dalla produzione di altri eventi dello stesso tipo. Per esempio "un ordine e' stato registrato" in un sistema di e-commerce potrebbe essere un evento di questo tipo
+- eventi correlati (nel tempo): il contenuto informativo di tali eventi non e' rappresentato dall'i-esimo evento in particolare, ma dallo stream di tutti gli eventi. Basti pensare ad un sensore di temperatura: e' molto piu' importante sapere se la "tendenza" della temperatura e' in discesa, per attivare la caldaia, piuttosto che sapere che in un particolare istante temporale, la temperatura rilevata era di 20.3 gradi
+
+In queste tre macro categorie si configurano i tre servizi principali che Azure mette a disposizione per la gestione dei messaggi:
+- [Azure Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/?WT.mc_id=AZ-MVP-5001625) per la gestione degli stream di eventi
+- [Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/?WT.mc_id=AZ-MVP-5001625) per la gestione degli eventi di qualsiasi forma e origine
+- [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview?WT.mc_id=AZ-MVP-5001625) per la gestione di scenari di business "critici" in cui si rivelano indispensabili le feature di un enterprise message broker completo
+
+Ora che abbiamo capito in che scenario ha senso pensare ad Event Grid, cerchiamo di capire un po' meglio cos'e': prima del suo avvento, tutti i servizi di Azure erano una serie di puntini, che potevano cooperare tra di loro, ma con modalita' completamente differenti tra di loro. Event Grid e' la lingua franca con cui mettere in comunicazione tutti questi servizi. La linea che permette di unire tutti i puntini.
+
+I [componenti principali](https://docs.microsoft.com/en-us/azure/event-grid/concepts?WT.mc_id=AZ-MVP-5001625) che caratterizzano Event Grid sono:
+- un evento, ovvero l'informazione che si vuol far "viaggiare" nel sistema
+- una sorgente (source), ovvero il componente che genera l'evento
+- un topic, ovvero l'endpoint a cui viene "inviato" l'evento
+- un handler, ovvero un componente interessato ad un particolare evento
+- una subscription, ovvero il collegamento tra un topic ed un componente interessato all'evento generato
+
+Nell'ecosistema di Azure, una _source_ puo' essere uno dei servizi esposti dalla piattaforma, come per esempio [Azure Container Registry](https://docs.microsoft.com/en-us/azure/event-grid/event-schema-container-registry?WT.mc_id=AZ-MVP-5001625) o [Azure App Configuration](https://docs.microsoft.com/en-us/azure/event-grid/event-schema-app-configuration?WT.mc_id=AZ-MVP-5001625). Un handler puo' essere per esempio una [Azure Function](https://docs.microsoft.com/en-us/azure/event-grid/handler-functions?WT.mc_id=AZ-MVP-5001625) o un [WebHook](https://docs.microsoft.com/en-us/azure/event-grid/handler-webhooks?WT.mc_id=AZ-MVP-5001625).  
```

Comment:
Mip iace l'idea di scrivere _source_ , in _italic_. Adotterei lo stesso stile anche per _handler_, _topic_ e così via (ed anche nella lista poco sopra).